components/organisms/RepositoriesTable/repositories-table.tsx

```diff
@@ -79,7 +79,7 @@ const RepositoriesTable = ({
   return (
     <section className="flex flex-col">
       {loading && <SkeletonWrapper height={50} count={10} radius={4} classNames="px-6 mt-2" />}
-      {error && <>An error has occured...</>}
+      {error && <>An error has occurred...</>}
```

Comment:
refactor (nit): this change isn't related to the MetricCard. Consider a separate PR for the spelling mistake.